apps/dashboard/app/views/files/index.html.erb

```diff
@@ -3,10 +3,10 @@ content_for :head do
   javascript_pack_tag 'files'
 end
 %>
+
 <div class="text-right">
 
-  <%# FIXME: open in terminal should be dropdown of all clusters (with default) %>
-  <a id="open-in-terminal-btn" class="btn btn-outline-dark btn-sm disabled" href="#"><i class="fas fa-terminal" aria-hidden="true"></i> Open in Terminal</a>
+  <%= render partial: 'shell', locals: { clusters: OodAppkit.clusters } %>
```

Comment:
You use the constant in the partial, so no need to pass the local "clusters" which is unused.

Reply:
Oh you do use clusters - probably should just use the `OodAppkit.clusters` directly in the partial view?